index.js

```diff
@@ -28,11 +30,38 @@ async function activate() {
 
   loadConfig(path);
 
-  if (!validateConfig() || !isTrackingAllowed()) {
+  if (!validateConfig()) {
     return;
   }
 
-  await trackUsage('vscode-extension', 'ACTIVATION');
+  if (isTrackingAllowed()) {
+    try {
+      await trackUsage(
```

Comment:
Out of the scope of this PR, but it seems like we could/should update `trackUsage` to only require the first two args here. The rest seem somewhat unnecessary.